src/common/tools/dev_server.ts

```diff
@@ -150,14 +150,17 @@ app.get('/out/**/*.js', async (req, res, next) => {
   const tsUrl = jsUrl.replace(/\.js$/, '.ts');
   if (compileCache.has(tsUrl)) {
     res.setHeader('Content-Type', 'application/javascript');
+    res.setHeader('Service-Worker-Allowed', '/');
```

Comment:
Is this needed? Seems like it's only needed for intercepting network requests which we don't need to do.

Reply:
I see. We still don't need a large scope though, which is specified in `navigator.serviceWorker.register` as `scope: '/'`. Can we do the other option:

> Adjust the scope

and just use the default scope which is minimal and seems like it should work fine.

Reply:
Also in particular we need this to work on gpuweb.github.io and we can't set headers there.

Reply:
You're right. I've removed `scope` and it works fine now.